### PR TITLE
Log version info

### PR DESCRIPTION
**Pull request by cdesiniotis**
```
time="2023-12-22T21:08:09Z" level=info msg="version: v0.2.4, commit: f085985-dirty"
```

### CODE REVIEW

File: versions.mk [hunk 1]
```diff
@@ -19,3 +19,5 @@ vVERSION := v$(VERSION:v%=%)
 CUDA_VERSION := 12.2.2
 
 GOLANG_VERSION := 1.20.4
+
```
Comment (elezar) [inline versions.mk:22]: Do we want the full SHA? (See container-toolkit)

Comment (cdesiniotis) [reply versions.mk:22]: Sure. I have updated to using the full sha for consistency.

File: deployments/container/Dockerfile.ubi8 [hunk 2]
```diff
@@ -39,16 +39,15 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
...
+RUN make PREFIX=/artifacts cmds
 
 
 FROM nvcr.io/nvidia/cuda:${CUDA_VERSION}-base-${BASE_DIST}
 
 ENV NVIDIA_VISIBLE_DEVICES=void
 
 COPY --from=build /artifacts/nvidia-vgpu-dm /usr/bin/nvidia-vgpu-dm
-COPY --from=build /artifacts/k8s-nvidia-vgpu-dm /usr/bin/k8s-nvidia-vgpu-dm
+COPY --from=build /artifacts/nvidia-k8s-vgpu-dm /usr/bin/nvidia-k8s-vgpu-dm
```
Comment (elezar) [inline deployments/container/Dockerfile.ubi8:50]: Just to confirm that no other components depend on this name specifically?

File: Makefile [hunk 3]
```diff
@@ -33,6 +33,7 @@ DOCKER_TARGETS := $(patsubst %, docker-%, $(TARGETS))
 .PHONY: $(TARGETS) $(DOCKER_TARGETS)
 
 GOOS := linux
+VERSION_PKG=$(MODULE)/internal/info
```
Comment (elezar) [inline Makefile:36]: Nit: Technically the use of this `VERSION_PKG` should also be part of this commit.

**Review by elezar — APPROVED** [review]
Minor comments, but LGTM.

File: deployments/container/Dockerfile.ubi8 (hunk 2 above)
Comment (cdesiniotis) [reply deployments/container/Dockerfile.ubi8:50]: Correct, no others depend on this name.

**Comment by tariq1890** [comment]
@cdesiniotis Can you rebase this? LGTM otherwise

**Review by tariq1890 — APPROVED** [review]

**Pull request merged**